src/abbfreeathome/api.py

```diff
@@ -220,6 +239,26 @@ async def set_datapoint(
 
         return True
 
+    async def virtualdevice(self, serial: str, data: dict[str, Any]):
+        """Create or modify a virtualdevice in the api."""
+        _schema = self.virtualdevice_main_schema.extend(
+            {vol.Required("properties"): self.virtualdevice_properties_schema}
+        )
+        try:
+            _schema(data)
+        except vol.MultipleInvalid as e:
+            raise vol.error.Invalid(e) from e
```

Comment:
My understanding is there's no need for a try/catch if you're just going to raise the exception anyway.

```suggestion
        _schema(data)
```